docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+services:
+    app:
+        build:
+            context: .
+        ports:
+            - "8000:8000"
+        volumes:
+            - ./:/app/
+            - my_media:/files/media
+            - my_static:/files/static
+        command: >
+            sh -c "python manage.py makemigrations &&
+            python manage.py wait_for_db
+            && python manage.py collectstatic --noinput
+            && python manage.py migrate &&
+                      python manage.py runserver 0.0.0.0:8000"
+        env_file:
+            - .env
+        depends_on:
+            - db
+
+    db:
+        image: postgres:14-alpine
+        restart: always
```

Comment:
The `restart: always` policy is set for the database service, which is good for ensuring the service restarts automatically. Consider applying a similar policy for the `app` service to improve resilience.